content/applications/finance/accounting/receivables/customer_payments/recording.rst

```diff
@@ -2,125 +2,120 @@
-   Exchange Loss/Gain posted at the time of reconciliation.
+   - The customer invoice or vendor bill should be in status :guilabel:`Posted` to register the
+     payment.
+   - When clicking on :guilabel:`Register payment`, you can select the amount to pay and therefore
+     make a partial or full payment.
+   - If your main bank account is put as :doc:`oustanding account
+     <../../bank/setup/outstanding_accounts>`, and the payment is made in Odoo (not related to a
+     bank statement), invoices and bills are directly registered in the status :guilabel:`Paid`.
+   - If you unreconcile a payment, it still appears in your books but is no longer linked to the
+     invoice.
+   - If you (un)reconcile a payment in a different currency, a journal entry is created to post the
+     Currency Exchange Loss/Gain (reversal) amount.
+   - If you (un)reconcile a payment and an invoice having cash basis taxes, a journal entry is
+     created to post the Cash Basis Tax (reversal) amount.
 
-Payments not tied to an invoice
-===============================
-
-Registering a payment
----------------------
-
-In the Accounting application, you can create a new payment from the
-Sales menu (register a customer payment) or the Purchases menu (pay a
-vendor). If you use these menus, the payment is not linked to an
-invoice, but can easily be reconciled on an invoice later on.
-
-.. image:: recording/recording02.png
-  :align: center
-
-When registering a new payment, you must select a customer or vendor,
-the payment method, and the amount of the payment. The currency of the
-transaction is defined by the payment method. If the payment refers to a
-document (sale order, purchase order or invoice), set the reference of
-this document in the memo field.
-
-Once confirmed, a journal entry will be posted reflecting the
-transaction just made in the accounting application.
+.. seealso::
+   - :doc:`../../bank/reconciliation/use_cases`
 
-Reconciling invoice payments
-----------------------------
+Registering payments not tied to an invoice or a bill
+=====================================================
 
-The easiest way of reconciling a payment with an invoice is to do so on
-the invoice directly.
+When a new payment is registered via the menu :menuselection:`Customers or Vendors --> Payments`, it
+is not directly linked to an invoice or a bill. The account receivable or the account payable have
+to be matched manually  with the related invoice or bill.
 
-When validating a new invoice, Odoo will warn you that an outstanding
-payment for this customer or vendor is available. In this case, you can
-reconcile this payment to the invoice near the totals at the bottom,
-under "Outstanding Payments".
+Matching invoices and bills with payments
+-----------------------------------------
 
-.. image:: recording/recording03.png
-  :align: center
+When validating a new invoice or a bill, a blue banner appears when there is an outstanding payment
```

Comment:
```suggestion
When validating a new invoice or bill, a blue banner appears when there is an outstanding payment
```
Tbh it doesn't seem that clear to me... I know what you are referring to but perhaps explain the condition to trigger that banner (that it only appears when Odoo detects a correlating amount and partner)